docs/install_guides/installing-openvino-from-archive-macos.md

```diff
@@ -30,60 +30,63 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
+   ```
+   sudo ln -s openvino_2022.2.0.7713 openvino_2022
    ```
-   where the `<destination_dir>` is the directory that you extract OpenVINO files to. You're recommended to set it as `/opt/intel/`.
-   The standard OpenVINO `INSTALL_DIR` referenced in this document is `/opt/intel/openvino_<version>`.
+   > **NOTE**: If you have already installed a previous release of OpenVINO 2022, a symbolic link to the `openvino_2022` folder may already exist. Remove the previous link with `sudo rm openvino_2022`, then re-issue the previous command.
 
-For simplicity, you can create a symbolic link to the latest installation: `/opt/intel/openvino_2022/`. 
+Congratulations, you finished the installation! The `/opt/intel/openvino_2022` folder now contains the core components for OpenVINO. If you used a different path in Step 2, you will find the `openvino_2022` folder there.
 
-The core components are now installed. Continue to the next section to configure environment.
+The path to the `openvino_2022` directory is also referred as `<INSTALL_DIR>` throughout the OpenVINO documentation.
 
 ### <a name="set-the-environment-variables"></a>Step 2: Configure the Environment
 
-You must update several environment variables before you can compile and run OpenVINO™ applications. Set environment variables as follows:
+You must update several environment variables before you can compile and run OpenVINO applications. Open a terminal window and run the `setupvars.sh` script as shown below to temporarily set your environment variables. If your <INSTALL_DIR> is not `/opt/intel/openvino_2022`, use the correct one instead.
 
 ```sh
-source <INSTALL_DIR>/setupvars.sh
-```
+source /opt/intel/openvino_2022/setupvars.sh
+```  
 
-If you have more than one OpenVINO™ version on your machine, you can easily switch its version by sourcing `setupvars.sh` of your choice.
+If you have more than one OpenVINO™ version on your machine, you can easily switch its version by sourcing the `setupvars.sh` of your choice.
 
-> **NOTE**: You can also run this script every time when you start new terminal session. Open `~/.bashrc` in your favorite editor, and add `source <INSTALL_DIR>/setupvars.sh`. Next time when you open a terminal, you will see `[setupvars.sh] OpenVINO™ environment initialized`. Changing `.bashrc` is not recommended when you have many OpenVINO™ versions on your machine and want to switch among them, as each may require different setup.
+> **NOTE**: The above command must be re-run every time the terminal is opened. To set up Linux so it automatically runs the command every time a new terminal is opened, open `~/.bashrc` in your favorite editor and add `source /opt/intel/openvino_2022/setupvars.sh` after the last line. Next time when you open a terminal, you will see `[setupvars.sh] OpenVINO™ environment initialized`. Changing `.bashrc` is not recommended when you have multiple OpenVINO™ versions on your machine and want to switch among them.
```

Comment:
Linux => macOS? Also, macOS has different terminal `zsh` (at least latest versions) and we should use `~/.zshrc` rather than `~/.bashrc`